packages/rrweb/src/replay/index.ts

```diff
@@ -571,10 +629,7 @@ export class Replayer {
     this.wrapper = document.createElement('div');
     this.wrapper.classList.add('replayer-wrapper');
     this.config.root.appendChild(this.wrapper);
-
-    this.mouse = document.createElement('div');
-    this.mouse.classList.add('replayer-mouse');
-    this.wrapper.appendChild(this.mouse);
+    this.createPointer(-1);
```

Comment:
this shows an inactive mouse in the top left corner when all the touch events have pointerIds (since -1 is never used in that case) - we need to hide the mouse in this case somehow

Reply:
Oof right, let's remove it then.